### ISSUE TITLE
Node.js Foundation Technical Steering Committee (TSC) Meeting 2017-09-20

### ISSUE THREAD

**Issue by mhdawson** [nodejs/TSC#359]
## Time

**UTC Wed 20-Sep-2017 12:00 (12:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 20-Sep-2017 05:00 (05:00 AM) |
| US / Mountain | Wed 20-Sep-2017 06:00 (06:00 AM) |
| US / Central  | Wed 20-Sep-2017 07:00 (07:00 AM) |
| US / Eastern  | Wed 20-Sep-2017 08:00 (08:00 AM) |
| Amsterdam     | Wed 20-Sep-2017 14:00 (02:00 PM) |
| Moscow        | Wed 20-Sep-2017 15:00 (03:00 PM) |
| Chennai       | Wed 20-Sep-2017 17:30 (05:30 PM) |
| Hangzhou      | Wed 20-Sep-2017 20:00 (08:00 PM) |
| Tokyo         | Wed 20-Sep-2017 21:00 (09:00 PM) |
| Sydney        | Wed 20-Sep-2017 22:00 (10:00 PM) |

Or in your local time:
* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Technical%20Steering%20Committee%20(TSC)+Meeting+2017-09-20&iso=20170920T12
* or http://www.wolframalpha.com/input/?i=12pm+UTC%2C+Sep+20%2C+2017+in+local+time

## Links

* **Minutes Google Doc**: <https://docs.google.com/document/d/1oMS1G-Tpgx8Ha8FqCb1XCLXSu5yZ6AjO1HmB1clUPaU/edit#>
* Previous minutes: <https://docs.google.com/document/d/1tOeEpEz2Qf7ybjsoFXRrMSpNrEcKIIfnc3eqDYeP0zU>_

## Agenda

Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/TSC
* Regular WG Status updates in TSC Meetings [#352](https://github.com/nodejs/TSC/issues/352)
* Authorise expenditure of $535.00 USD for Microsoft Authenticode signing certificate [#347](https://github.com/nodejs/TSC/issues/347)
* Candidate "areas" for TSC and Community Committee  [#278](https://github.com/nodejs/TSC/issues/278)

### nodejs/security-wg

* CVE management process for Node.js [#33](https://github.com/nodejs/security-wg/issues/33)

## Invited

* Anna Henningsen @addaleax (TSC)
* Ben Noordhuis @bnoordhuis (TSC)
* Сковорода Никита Андреевич @ChALkeR (TSC)
* Colin Ihrig @cjihrig (TSC)
* Evan Lucas @evanlucas (TSC)
* Franziska Hinkelmann @fhinkel (TSC)
* Jeremiah Senkpiel @Fishrock123 (TSC)
* Fedor Indutny @indutny (TSC)
* James M Snell @jasnell (TSC)
* Joyee Cheung @joyeecheung (TSC)
* Matteo Collina @mcollina (TSC)
* Michael Dawson @mhdawson (TSC)
* Brian White @mscdex (TSC)
* Myles Borins @MylesBorins (TSC)
* Ali Ijaz Sheikh @ofrobots (TSC)
* Rod Vagg @rvagg (TSC)
* Shigeki Ohtsu @shigeki (TSC)
* Michaël Zasso @targos (TSC)
* Sakthipriyan Vairamani @thefourtheye (TSC)
* Trevor Norris @trevnorris (TSC)
* Rich Trott @Trott (TSC)
* Josh Gavant @joshgav (TSC)

### Observers
* Bradley Meck @bmeck (GoDaddy/TC39)
* Tracy Hinds @hackygolucky (Node.js Foundation Education Community Manager)
* Kaitlyn Barnard @kbarnard10 (Node.js Foundation Newsletter Curator)
* Mark Hinkle @mrhinkle (Node.js Foundation Executive Director)
* William Kapke @williamkapke (Node.js Community Board representative)

## Notes

The agenda comes from issues labelled with `tsc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Uberconference; participants should have the link & numbers, contact me if you don't.

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **https://www.youtube.com/c/nodejs+foundation/live** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

Many of us will be on IRC in #node-dev on Freenode if you'd like to interact, we have a Q/A session scheduled at the end of the meeting if you'd like us to discuss anything in particular. @nodejs/collaborators in particular if there's anything you need from the TSC that's not worth putting on as a separate agenda item, this is a good place for it.

**Comment by mcollina** [comment]
Let's remove https://github.com/nodejs/node/pull/12857 and https://github.com/nodejs/node/issues/445 from the TSC agenda. Let's wait the vote to finish off.

**Comment by mhdawson** [comment]
Also please review:

### nodejs/node

* Regression: Resolution algorithm collision [#14990](https://github.com/nodejs/node/issues/14990)
* path: deprecate internal _makeLong, replace [#14956](https://github.com/nodejs/node/pull/14956)
* build: update minimum kernel version to 3.10.0 [#14795](https://github.com/nodejs/node/pull/14795)
* Buffer.isEncoding regards an empty string as a valid encoding [#9654](https://github.com/nodejs/node/issues/9654)

### nodejs/TSC

* Form an Admin team / subcommittee [#328](https://github.com/nodejs/TSC/issues/328)
* Candidate "areas" for TSC and Community Committee  [#278](https://github.com/nodejs/TSC/issues/278)
* Approval to move node-heapdump into foundation [#257](https://github.com/nodejs/TSC/issues/257)
* create public nodejs/collaborators repo [#243](https://github.com/nodejs/TSC/issues/243)

**Comment by evanlucas** [comment]
I won’t be able to make it this week. I’m on vacation. Thanks!

**Comment by thefourtheye** [comment]
I just edited the OP to fix the links.

**Comment by jasnell** [comment]
Did the uberconference number change?

**Comment by jasnell** [comment]
(asking just to make sure, I thought I remembered some chatter about that changing)

**Comment by mhdawson** [comment]
@jasnell not as far as I know.

**Comment by Trott** [comment]
> Did the uberconference number change?

No, but it really ought to. What does the Board use for remote meetings and can we use that if it's something the Foundation pays for already?

**Comment by Trott** [comment]
Actually ignore my question here. I'll open a separate issue if there isn't one already that's more appropriate.

**Comment by jasnell** [comment]
Another tsc-review item: https://github.com/nodejs/build/issues/885

**Comment by fhinkel** [comment]
Probably won't be able to make it.

**Comment by mhdawson** [comment]
Minutes: https://github.com/nodejs/TSC/pull/361

**Comment by gibfahn** [comment]
Did nodejs/build#885 get missed?

**Comment by targos** [comment]
@gibfahn it is not labeled `tsc-agenda`.

**Comment by gibfahn** [comment]
Oh I see, so @jasnell linked from it in here to invite people to comment on the issue. In that case please do.
